SUMO-254678: Doc update Database apps migration V1 to V2 (#5009)
SumoLogic/sumologic-documentation: docs/integrations/microsoft-azure/sql-server.md

This section provides instructions for configuring a local file source to collect SQL Server ERRORLOG data, and a script source to collect SQL Server performance metrics. A sample log message is also provided.

### Step 1: Configure fields in Sumo Logic

> As part of the app installation process, the following fields will created by default:

Reviewer: ```suggestion
As part of the app installation process, the following fields will be created by default:
```

* `component`
* `environment`
* `db_system`
* `db_cluster`
* `db_cluster_address`
* `db_cluster_port`

> Additionally, if you are using SQL Server in the Kubernetes environment, these following additional fields will be created by default as a part of the app installation process:

Reviewer: ```suggestion
Additionally, if you are using SQL Server in the Kubernetes environment, the following additional fields will be created by default during the app installation process:
```

* `pod_labels_component`
* `pod_labels_environment`
* `pod_labels_db_system`
* `pod_labels_db_cluster`
* `pod_labels_db_cluster_address`
* `pod_labels_db_cluster_port`

> To learn more about setting up fields, refer to [Sumo Logic Fields](/docs/manage/fields).

Reviewer: ```suggestion
For information on setting up fields, see [Fields](/docs/manage/fields).
```

```
2. Sumo Logic Kubernetes collection will automatically start collecting logs from the pods having the annotations defined above.
3. Verify logs in Sumo Logic.

> <br/>**FER to normalize the fields in Kubernetes environments.** Labels created in Kubernetes environments automatically are prefixed with pod_labels. To normalize these for our app to work, we will have Field Extraction Rule automatically created for Database Application Components named as **AppObservabilitySQLServerDatabaseFER**.

Reviewer: ```suggestion
<br/>**FER to normalize the fields in Kubernetes environments.** Labels created in Kubernetes environments automatically are prefixed with pod_labels. To normalize these for our app to work, a Field Extraction Rule named **AppObservabilitySQLServerDatabaseFER** is automatically created for Database Application Components.
```
